modules/mobile/app/workers/mobile/v0/pre_cache_appointments_job.rb

```diff
@@ -11,14 +11,15 @@ class MissingUserError < StandardError; end
 
       def perform(uuid)
         return unless Flipper.enabled?(:mobile_precache_appointments)
-
+        # remove this
```

Comment:
Category: success/failure
Action: remove. this could be better accomplished with statsd